tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -848,6 +848,12 @@ public function dataFileAsserts(): iterable
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-3853.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6936-limit.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6927.php');
+
+		if (PHP_VERSION_ID < 80000) {
+			return;
+		}
+
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-5262.php');
```

Comment:
the cs-fixer is doing this ugly thing a lot, and I don't like it.

```suggestion
		if (PHP_VERSION_ID >= 80000) {
    		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-5262.php');
		}

		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6927.php');
```

to work arround such silly IF which might lead to the next person to append a new test-case which then accidentally only runs on php < 8, I am using a different construction

Reply:
Yeah, I didn't like it either but forgot to do something about it. Thanks!